sky/engine/web/Sky.cpp

```diff
@@ -49,6 +49,7 @@ namespace {
 void willProcessTask() {}
 
 void didProcessTask() {
+  TRACE_EVENT0("flutter", "RunMicrotasks");
```

Comment:
Please don't add a trace event here.  This program point is absurdly hot.  If you want a trace point here, add it after the branch that checks whether there actually are any microtasks.